File: code/java/titleToNumber.java
class Solution {
    public int titleToNumber(String s) {
        int[] A = new int[s.length()];
        for(int i=0;i<s.length();i++)
            A[i] = (int)Math.pow(26,s.length()-1-i);
        int res = 0;
        for(int i=0;i<s.length();i++)
            res += A[i] * ((int)s.charAt(i) - ((int)'A') + 1);
        return res;
    }
}

File: code/python/titleToNumber.py
class Solution:
    def titleToNumber(self, s: str) -> int:
        A = [pow(26,(len(s)-1-i)) for i in range(len(s))]
        count = 0
        
        for i in range(len(s)):
            count += A[i] * (ord(s[i]) - ord('A')+1)
        
        return count
